src/ui/public/state_management/state_storage/_tests__/state_hash.js

```diff
@@ -0,0 +1,56 @@
+import expect from 'expect.js';
```

Comment:
I think the folder is missing an extra `_`. Perhaps add one for consistency? (`__tests__`)